src/js/post-911-gib-status/containers/IntroPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+
+import { getServiceUp } from '../actions/post-911-gib-status';
+import { SERVICE_UP_STATES } from '../utils/constants';
+
+export class IntroPage extends React.Component {
+  constructor(props) {
+    super(props);
+    // Make the api request
+    this.props.getServiceUp();
+  }
+
+
+  getContent() {
+    let content;
+    switch (this.props.serviceUp) {
+      case SERVICE_UP_STATES.unrequested: {
+        // This is never actually even seen
+        content = (<div></div>);
+        break;
+      }
+      case SERVICE_UP_STATES.pending: {
+        // TODO: Change the loading message
+        content = <LoadingIndicator message="Please wait while we check if the tool is available."/>;
+        break;
+      }
+      case SERVICE_UP_STATES.up: {
+        // TODO: Determine whether h2 is right--accessibility-wise, it is, but it's larger than the design
```

Comment:
what's the status on this?

Reply:
Good question. I don't think I ever got a solid answer on this one.

@goldenmeanie Do you have an official answer?

Reply:
Go with correct hierarchy